packages/core/src/Timer.ts

```diff
@@ -0,0 +1,92 @@
+type Tick = {
+    // Result of Date API. Unix timestamp in ms.
+    timestamp: number,
+    // Result of performance.now API. Timestamp in ms (with microsecond precision)
+    // since JS context start.
+    timestamp_rn: number | null
+}
```

Comment:
nit: what about `type Timestamp { unix: number, rn: number | null }` instead of `Tick`?